assets/src/domain/blocks/event-attendees/controls/SelectEvent.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { SelectControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+import { AttendeesEditProps } from '../types';
+import useEvents from '../hooks/useEvents';
+import { buildEntitySelectOptions } from '@sharedServices/utils/entityList';
+import { SelectControlProps } from '../../components/types';
+
+const SelectEvent: React.FC<AttendeesEditProps> = ({ attributes, setAttributes }) => {
+	const { event } = attributes;
+
+	const { data, loading, error } = useEvents();
+
+	const options = buildEntitySelectOptions(data?.espressoEvents?.nodes || [], loading, error);
```

Comment:
maybe setting default value can be done in `buildEntitySelectOptions`

Reply:
It's not about the default value, it's about passing an array which the utility function can loop over.

Reply:
I think Vadim means to set the default value in `buildEntitySelectOptions()` instead of having to add `|| []` to every one of those function calls.

I'm guessing you have done it this way to satisfy TypeScript.

Another option might be to set a var beforehand like:

```js
const events = data?.espressoEvents?.nodes || [];
const options = buildEntitySelectOptions(events, loading, error);
```